ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoTransactionsSpec.scala

```diff
@@ -135,40 +135,13 @@ private[dao] trait JdbcLedgerDaoTransactionsSpec extends OptionValues with Insid
             startExclusive = from,
             endInclusive = to,
             filter = Map(alice -> Set.empty, bob -> Set.empty, charlie -> Set.empty),
-            pageSize = 100,
             verbose = true,
           ))
     } yield {
       comparable(result) should contain theSameElementsInOrderAs comparable(lookups)
     }
   }
 
-  it should "return the same result regardless of the page size" in {
```

Comment:
Was this moved somehwere else?

Reply:
Not really, the initialization of `eventsPageSize` has been moved to the data access object, which is initialized by the test suite. I would need a second data access object to test this. I thought of ways in which I could make this happen but then I realized I was basically testing `PaginatingAsyncStream` and `groupContiguous`. Let me know if you're ok to drop this.